docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/index.rst

```diff
@@ -1,1213 +1,152 @@
+These percentiles, also known as propensities, are applied to different distributions as the simulant changes age group,
+which means that a simulant's risk exposure in one age group and the next are perfectly auto-correlated.
 
-where we have written :math:`e_1` and :math:`e_2` as “standardized”
-exposures, after scaling and shifting so that the TMREL is 0.0 and a one
-unit increment gives the relative increase in disease burden reported by
-GBD.
+For example, a simulant might have their propensities for height and weight sampled at birth from a
+multivariate uniform distribution with correlation.
+If the simulant receives a propensity of 0.85 (85th percentile) in height and 0.80 (80th percentile) in weight,
+they are initially assigned the height and weight values at those percentiles among newborns.
+But when they age into the next age group, they are assigned the values at those same percentiles in their new
+age group, and so on throughout their life.
+In this way, their height and weight can change over time, but the correlation introduced in the initial sampling
+of propensities remains.
 
-With the log-linear assumption, we can calculate the PAF from first
-principles as a double integral (below :math:`p_{e_1,e_2}` denotes the
-joint probability density of exposure level :math:`(e_1,e_2)` in the
-population:
+.. todo::
+  Be more exact about the properties of the propensity correlation approach.
+  I believe that it only exactly maintains the correlation coefficient with some distributional assumptions.
 
-..
-  :raw-latex:`\begin{align*}
-  i_{\text{GBD}} &= \int_{e_1} \int_{e_2} i_{e_1, e_2} p_{e_1,e_2} de_1 de_2\\
-  &= \int_{e_1} \int_{e_2} i_0 \cdot \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2}
-  p_{e_1,e_2} de_1 de_2\\
-  &= \int_{e_1} \int_{e_2} \left(1-\text{PAF}\right) \cdot i_{\text{GBD}} \cdot \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2}
-  p_{e_1,e_2} de_1 de_2\\
-  \end{align*}`
+Risks with effects on the same outcome
+--------------------------------------
 
-.. math::
-  :nowrap:
+When we introduce correlation between two risk factors that have effects on the same outcome, we have to
+be especially careful about double-counting issues.
 
-  \begin{align*}
-  i_{\text{GBD}} &= \int_{e_1} \int_{e_2} i_{e_1, e_2} p_{e_1,e_2} de_1 de_2\\
-  &= \int_{e_1} \int_{e_2} i_0 \cdot \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2}
-  p_{e_1,e_2} de_1 de_2\\
-  &= \int_{e_1} \int_{e_2} \left(1-\text{PAF}\right) \cdot i_{\text{GBD}} \cdot \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2}
-  p_{e_1,e_2} de_1 de_2
-  \end{align*}
+The population-attributable fractions (PAFs) must be calculated correctly,
+or the population-level incidence of the outcome will
+not validate to GBD.
+**The usual Vivarium strategy to calculate joint PAFs (PAFs of more than one risk factor on an outcome)
+is biased when the risk factors are correlated.**
```

Comment:
I think it would be worthwhile to state that the "usual Vivarium strategy" is the same one the GBD uses, so GBD overestimates PAFs when risk factors are positively correlated.

Reply:
Also, I think it would be helpful to specifically reference the "multiplicative PAF" for the usual vivarium strategy for calculating PAFs when multiple risks affect the same outcome in case the reader doesn't already have that understanding.

Reply:
I know there was once a proof of this idea, but is there a 1 sentence intuitive "why" for this concept?

Reply:
> I know there was once a proof of this idea, but is there a 1 sentence intuitive "why" for this concept?

@SylLutze I see @alibow  added an explanation below, and I think @zmbc had a good intuitive explanation yesterday at the documentation session.

The mathematical proof that correlation induces bias in the PAF is also pretty simple and allows us to easily deduce the direction of bias:

By definition, the relative risks are positively correlated if their covariance is positive, which [means precisely that](https://en.wikipedia.org/wiki/Covariance#Definition) `E[RR(r1) * RR(r2)] > E[RR(r1)] * E[RR(r2)]`. Therefore,
```
PAF_joint = 1 - 1 / E[RR(r1) * RR(r2)]
          > 1 - 1 / ( E[RR(r1)] * E[RR(r2)] )
          = PAF_mult,
```
since `1 - 1/x` is a monotonically increasing function of `x` for `x>0`. That is, the multiplicative PAF (PAF_mult) underestimates the true joint PAF (PAF_joint) when the risks are positively correlated. Similarly, switching the inequality shows that PAF_j < PAF_m precisely when RR(r1) and RR(r2) are are negatively correlated, and PAF_joint = PAF_mult when the relative risks are uncorrelated.

We can translate the mathematical proof into more natural language as follows: Since we assumed that the relative risks of r1 and r2 are multiplicative (i.e., RR(r1,r2) = RR(r1)*RR(r2)), if r1 and r2 are positively correlated, then the average joint relative risk of (r1,r2) is greater than the product of the average relative risks of r1 and r2, which implies that the joint PAF is greater than the multiplicative PAF. The only remaining intuition to fill in for this explanation is why positive correlation of two random variables X and Y is equivalent to the inequality E(XY) > (EX)(EY), if we don't take this as the definition of positive correlation; this seems related to what Ali was getting at in her explanation.

Reply:
Thanks! Yeah, this is helpful for me to understand better. I think similarly it would be helpful to add this to the page for future folks.